canonical-web-and-design/www.ubuntu.com
# 5031 iot devops landing and takeover

## Done

Add IoT DevOps WP takeover and landing page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/) and [/engage/iot-devops](http://0.0.0.0:8001/engage/iot-devops)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Make sure it complies with the [brief](https://docs.google.com/document/d/1imuKA06xisHD5RURurRhiJuoVeQ3qGojpOVX6odPxR0/edit?ts=5caa27c6#)
- Make sure it complies with the [design](https://github.com/canonical-web-and-design/www.ubuntu.com/issues/5023)


## Issue / Card

Fixes #5031
